### ISSUE TITLE
How to export a proper NFO

### ISSUE THREAD

**Issue by alxlaxv** [MediaArea/MediaInfo#281]
Hello there, 
I have built a custom Confignon to export informations about my movie but here is my problem:
- The export is contains ASCII characters I need
- The export is done in .txt

When I rename the file in .nfo, all ASCII are broken. I have to manually Save As MSDOS Info file / .nfo and then open it and paste my ASCII inside to finally save and have a proper NFO.

Is there any way to do that automatically?

Thanks a lot for your help and contribution. Great app!

**Comment by JeromeMartinez** [comment]
On Windows, I personally use (command line) `mediainfo --LogFile=MyLogFile.txt MyFile.ext` in order to store in UTF-8, which is the best [locale](https://en.wikipedia.org/wiki/Locale_(computer_software)) IMO for transferring documents.
I can not talk about "NFO" as I have no idea about what (which locale) you/they expect. you say "proper NFO", please provide a link to the specification of this "format" so I can understand what "proper NFO" technically means.

**Comment by alxlaxv** [comment]
Thanks for your feedback, I will give it a try.
The problem is that windows seems to change the encoding when renaming the .txt to .nfo and the GUI don’t care about the extension I put.
Can I add on top of that the —output option to choose a custom csv template?

**Comment by alxlaxv** [comment]
I have made some tests and here is the situation now.
On Windows, I cant correctly generate the output with my custom template in CLI with
`"!mediainfo_path!" "--output=file://!output_extension!" "%%i" > !shortpath!.txt`
but as soon as I change that to .nfo, the file is not saved anymore in UTF-8 but in OEM-US and all my ASCII are broken.

So now, I'm trying to do the same on Linux with the CLI command: 
`./mediainfo --Output="/volume1/template.csv" --LogFile="/volume1/output.nfo" "/volume1/file.mkv"`
but I only get a simple output file, my template is not used.

Can you please help on that?

Thanks

**Comment by JeromeMartinez** [comment]
> The problem is that windows seems to change the encoding when renaming the .txt to .nfo 

File renaming is just file renaming, maybe your tool (not one from Windows by default, .nfo is by default associated to "System Information" program) does not like UTF-8?

> the file is not saved anymore in UTF-8 but in OEM-US

Please provide this file in OEM-US, so I can check it. Because I don't see how it could be OEM-US.

> but I only get a simple output file, my template is not used.

You forgot "file://" in your command.

**Comment by alxlaxv** [comment]
> You forgot "file://" in your command.
Doesn’t seem to work with that on Linux. Please advice!
Thanks!

**Comment by alxlaxv** [comment]
It was a typo error. My bad.
So I got it properly displayed in the console and in the .txt file here: https://send.firefox.com/share/b0e1d9a084
But if I save the file as .nfo, here is the result: https://send.firefox.com/share/be9c7f3b37

**Comment by JeromeMartinez** [comment]
wrong links.

**Comment by alxlaxv** [comment]
Edited.

**Comment by JeromeMartinez** [comment]
As expected both files are same (same file hash and so on). They are both in UTF-8. None is in OEM-US.
.nfo is not supported by default by Windows, so I tested with notepad++, .txt is open automatically in UTF-8, I had to configure manually for .nfo (open by default in local code page) for UTF-8 (I change nothing in the file, I just say to notepad++ that the file is encoded in UTF-8) and it is fine.

Summary: renaming does not change the encoding of the file but may change the behavior of your reading tool. Not a MediaInfo issue.
